src/controllers/graphql/types/post/postType.js

```diff
@@ -22,6 +22,7 @@ type Post {
   tag3: String
   tag4: String
   tag5: String
+  deleted: Boolean!
```

Comment:
Let's prefix booleans with `is` or `has`

Reply:
changed to 'isDeleted'